### DISCUSSION TITLE
Add a countdown timer in the multiplayer UI when an "!mp timer" is created

### DISCUSSION THREAD

**Discussion by DigitalHypno** [ppy/osu#31002] [Ideas]
Currently, when an "!mp timer" is made, you can only know how much time is remaining when BanchoBot sends an update in chat. It would be nice if osu!lazer had a constant display of the timer somewhere in its multiplayer UI. This is often useful during tournament matches where players want to check how much time is left before they need to pick the next map or enter the lobby after a map is picked, especially when they weren't in the lobby when the timer was made.

A similar but different suggestion is to add a new !mp command that lets you check the time remaining on the timer, something like "!mp checktimer". This would work in stable, meaning tournaments today would be able to use it as well. If the latter suggestion seems possible, I can make a separate thread for it.

**Comment by Layendan** [comment]
Lazer has a timer already implemented in the UI displaying the time remaining.
![image](https://github.com/user-attachments/assets/862d6b24-122d-4499-9265-a3b27365f26f)

For the standard side of things, that version is feature locked so new features won't be implemented.

**Reply by cl8n** [reply]
OP is also asking for general timers to behave like this, not just map start timers. 

afaik general timers are not supported at all in lazer yet, though. related to https://github.com/ppy/osu/discussions/16392
